hr_timesheet_day_week/models/account_analytic_line.py

```diff
@@ -0,0 +1,19 @@
+from odoo import api, fields, models
+
+class AccountAnalyticLine(models.Model):
+    _inherit = "account.analytic.line"
+    days_week = fields.Selection([
```

Comment:
IMO this is a better name:
```suggestion
    day_week = fields.Selection([
```